main.go

```diff
@@ -75,8 +75,8 @@ func main() {
 		},
 		cli.StringFlag{
 			Name:        "add-local",
-			Usage:       "Add local cluster (true, false, auto)",
-			Value:       "auto",
+			Usage:       "Add local cluster (true, false)",
+			Value:       "true",
```

Comment:
I just want to confirm that if for some reason "auto" is passed as a value Rancher will treat that as "true."  For example an upgrade of a chart that had is set to "auto" might still retain the value of "auto" in the new version.

Reply:
Ahh, good point. I need to update the `localClusterEnabled` function to allow `auto` or `true`. Right now it only accepts true and otherwise sets it to false.

Reply:
Updated